### ISSUE TITLE
[Survival Beta] 24w09c - [Suggestion] Mined rocks shouldn't have collision with player character.

### ISSUE THREAD

**Issue by dean-proca** [tsunamayo/Starship-EVO#5613]
### Short Description of the issue:

Mined rocks can throw player character into the air due to collision.

### Steps to reproduce the issue:

Depending on mined rocks angle of approach to player, if they approach from under the player.
Please see video at 9:15, 11:54. Link in video description to times of interest.
Long video requires some time for YouTube to process to HD.

### Environment
- Starship EVO version: 24w09c
- Graphics Card: RTX 3080ti

### Additional information:

https://www.youtube.com/watch?v=Xa05IBeTY38

Please also include the `Player.log` file. You can find the log file here: `%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\`

If you issue concerns a particular design please include any blueprint that could help to reproduce the issue.
Blueprint folder is over at:
`%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\Save_Data\Blueprints\`
You will need to zip the folder to upload it. 
To do so Right click and choose [Send to -> compressed zip file] then drag the .zip onto your report to upload it.

**Comment by surajnairpv** [comment]
It's not just the angle, this issue usually happens when your mining and once the rocks spawn you open up your inventory before it reaches you. The absorption kinda deactivates and the collision to player activates when the inventory is open.

**Comment by dean-proca** [comment]
> It's not just the angle, this issue usually happens when your mining and once the rocks spawn you open up your inventory before it reaches you. The absorption kinda deactivates and the collision to player activates when the inventory is open.

Thanks for the clarification.

**Comment by DrizztCode** [comment]
> It's not just the angle, this issue usually happens when your mining and once the rocks spawn you open up your inventory before it reaches you. The absorption kinda deactivates and the collision to player activates when the inventory is open.

i've never had it happen due to opening my inventory (since i rarely open my inventory when mining, usually making sure the slot for the ore is on the same bar as my mining laser)

but i have had it happen quite a lot

i wonder if it is more to do with the laser deforming the terrain and that causing the physics to get confused

i can see how it could be the spawned rocks too - since most of these things are due to the player and an object trying to occupy the same space - although usually the terraim deformation tends to get the player stuck

either way - gonna need this fixed or to get some shock absorbing boots if we get player fall damage soon lol :-)
(is not unreasonable to have an auto jetpack feature that does a quick burst to null a player fall - could be set to use a little bit of hydrogen - with the player having a small suit reservoir)

**Comment by Gallt** [comment]
I'll add: When you right-click to pick up blocks from somthing you're building, it can knock it around even if its attached to a starter block making it very difficult to build. The collision on objects in general is too high.

**Comment by surajnairpv** [comment]
The starter block has its issues #5541 and #5617
Tsuna has thoughts on removing the feature.

**Comment by tsunamayo** [comment]
Yes this more has to do with the character locomotion than the rock tbh! I know it was on my to do list for a while, it is just really tricky...
